spouts/rss/feed.php

```diff
@@ -212,7 +212,7 @@ public function getId() {
      */
     public function getTitle() {
         if ($this->items !== false && $this->valid()) {
-            return @current($this->items)->get_title();
+            return htmlspecialchars_decode(@current($this->items)->get_title());
```

Comment:
[RSS specs](https://cyber.harvard.edu/rss/rss.html#hrelementsOfLtitemgt) suggest HTML can only appear within `description` element. The following feed was broken by the PR:

http://www.vzhurudolu.cz/rss